pkg/agent/openflow/multicast.go

```diff
@@ -196,6 +202,25 @@ func (f *featureMulticast) initGroups() []binding.OFEntry {
 	return nil
 }
 
+func (f *featureMulticast) multicastForwardFlexibleIPAMFlows(table binding.Table) []binding.Flow {
```

Comment:
I see the 3rd flow in `multicastRemoteReportFlows` does similar thing, it matches traffic from tunnel port but resubmits to `MulticastEgressRule`. What's the difference here? Why we use `MulticastEgressRule` in one case but `MulticastRouting` in another case.
Could the flows be unified and code be less redundant in some way?

Reply:
Updated the code in `multicastForwardFlexibleIPAMFlows,` making it less redundant.

I think flows in `multicastForwardFlexibleIPAMFlows` should also be resubmitted to the first table of the. Although The Flows in this function will not match the ingress Tables from our current implementation, if we implement the multicast ingress networkpolicy later, resubmitting to the `MulticastRouting` table directly may make the NP not work.

Reply:
Updated `multicastForwardFlexibleIPAMFlows,` making it concise.
 
I will change to flows in `multicastForwardFlexibleIPAMFlows` to resubmit to the first table of the Multicast pipeline as well because 1. it should not match multicast egress rules by our design, the effect is the same as we resubmit to the MulticastRouting table. 2. cause less confusion to code readers.

Reply:
I meant unifying these flows in some way. It seems hard to track how the flows are installed in different mode. In encap mode, the classifier is installed via `InstallMulticastRemoteReportFlows` but is via `InstallMulticastFlexibleIPAMFlows` in bridge mode, the naming style are quite different, one is named by purpose, another named by mode.
And I'm looking for how noEncap mode is handled.